### DISCUSSION TITLE
Minor error in the message token mail input (2FA)

### DISCUSSION THREAD

**Discussion by Nyxtorm** [dani-garcia/vaultwarden#1525] [Q&A]
### Subject of the issue

During 2FA, the message for the length of the mail token does not correspond to the value chosen for `EMAIL_TOKEN_SIZE`

### Deployment environment

```
* Bitwarden_rs version: v1.19.0-551810c4
* Web-vault version: v2.19.0
* Running within Docker: false
* Internet access: true
* Uses a proxy: false
* DNS Check: true
* Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Clients used: web
* Reverse proxy and version: nginx/1.19.8
```

### Steps to reproduce
1- Connect with the 2FA, choosing the option by mail
2- Compare the input message with the value of parameter `EMAIL_TOKEN_SIZE`

### Expected behaviour
This message should get its value from the `EMAIL_TOKEN_SIZE` parameter

### Actual behaviour
Its value is fixed at 6

### Troubleshooting data

Here, `EMAIL_TOKEN_SIZE` is set to 10

![image](https://user-images.githubusercontent.com/20800786/111969227-493b1d00-8afa-11eb-89f5-50691eed2440.png)

Sorry for my bad English.

**Comment by BlackDex** [comment]
That is not something we can change dynamically. It is build in into the the bitwarden web-vault.
See:
 * https://github.com/bitwarden/web/blob/1ea8762eeb6b43871e2101995e12668a6380fb45/src/locales/fr/messages.json#L651
 * https://github.com/bitwarden/web/blob/1ea8762eeb6b43871e2101995e12668a6380fb45/src/locales/en/messages.json#L651

**Reply by BlackDex** [reply]
Also, Bitwarden.com only supports 6, we made it dynamic.

**Reply by Nyxtorm** [reply]
Thanks for those details. This could be improved with more generic messages for example, like:

`Saisissez le code de vérification qui vous a été envoyé par e-mail à $EMAIL$.`
`Enter the verification code that was emailed to $EMAIL$.`

But that would require changing all the available languages.

In the meantime, this is only visual and it does not block the use of Bitwarden at all! 😉

**Reply by BlackDex** [reply]
And, since that is not our source code, but upstream bitwarden.com's, we do not want to change those items.
